plugins/ui/edgy.nix

```
@@ -182,84 +181,82 @@ helpers.neovim-plugin.mkNeovimPlugin config {
       } "Global window options for edgebar windows.";
 
       # This option accepts an attrs or a lua string.
-      # Hence, we use `mkOption` to convert the string to raw lua in `apply`.
-      keys = mkOption {
+      # Hence, we convert the string to raw lua in `apply`.
+      keys = helpers.mkNullOrOption' {
         type = with helpers.nixvimTypes; attrsOf (either strLuaFn (enum [ false ]));
-        default = { };
+        default = null;
         apply = mapAttrs (_: v: if isString v then helpers.mkRaw v else v);
```

I believe this is causing the CI failure, since I made the type nullable...
```suggestion
        apply =
          x:
          if x == null then null
          else mapAttrs (_: v: if isString v then helpers.mkRaw v else v);
```
Or just revert to non-nullable and call `helpers.pluginDefaultText` directly\
or implement the `mkOption` wrapper mentioned in the PR description.